package.json

```
@@ -12,7 +12,7 @@
     "url": "https://github.com/cypress-io/cypress-realworld-app/issues"
   },
   "engines": {
-    "node": ">=12.22.0"
+    "node": ">=16.16.0 < 18.0.0"
```

```suggestion
    "node": "^16.16.0"
```

`yarn dev` fails with `Error: error:0308010C:digital envelope routines::unsupported` on Node.js >=`17` so only `^16.16.0` is allowed.